src/test/scala/org/broadinstitute/dsde/firecloud/service/StatusServiceSpec.scala

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.dataaccess.HttpAgoraDAO
+import org.broadinstitute.dsde.firecloud.{Application, FireCloudConfig}
+import org.broadinstitute.dsde.firecloud.mock.MockUtils
+import org.broadinstitute.dsde.firecloud.model.{SubsystemStatus, SystemStatus}
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.impSystemStatus
+import org.broadinstitute.dsde.firecloud.webservice.StatusApiService
+import org.mockserver.integration.ClientAndServer
+import org.mockserver.integration.ClientAndServer.startClientAndServer
+import org.mockserver.model.HttpError
+import org.mockserver.model.HttpRequest.request
+import spray.http.StatusCodes.{InternalServerError, OK}
+import spray.routing.HttpService
+import spray.httpx.SprayJsonSupport._
+
+
+/**
+  * Created by anichols on 4/13/17.
+  */
+class StatusServiceSpecMockServer extends BaseServiceSpec with HttpService with StatusApiService {
+
+  def actorRefFactory = system
+
+  // Typically, the BaseServiceSpec provides an `app: Application` member that has all Mock DAOs.
+  // Here, we want to send requests through the Agora DAO to the Mock Server, so create a new Application
+  // with the HTTP DAO instead of Mock.
+  val customApp = new Application(new HttpAgoraDAO(FireCloudConfig.Agora), googleServicesDao, ontologyDao, rawlsDao, searchDao, thurloeDao)
+  val statusServiceConstructor: () => StatusService = StatusService.constructor(customApp)
+
+  var agoraServer: ClientAndServer   = startClientAndServer(MockUtils.methodsServerPort) // Agora = methods
+
+  override def afterAll(): Unit = {
+    agoraServer.stop()
+  }
+
+  agoraServer.when(
+    request().withMethod("GET").withPath("/status")
+  ).respond(
+    org.mockserver.model.HttpResponse.response()
+      .withHeaders(MockUtils.header)
+      .withBody("""{ "status": "down", "message": ["Agora is down"] }""")
+  )
+
+  "StatusService indicates Agora failure when \"down\" response" in {
```

Comment:
no need to change the PR, but for your edification: you can use triple-quotes to avoid escaping inner quotes:
```
  """StatusService indicates Agora failure when "down" response""" in {
```